src/sub_platforms/sql_opt/videx/scripts/videx_build_env.py

```diff
@@ -176,10 +176,8 @@ def parse_connection_info(info):
                                                                   raise_error=True,
                                                                   )
     elif args.fetch_method == 'sampling':
-        # We will introduce the sampling-based method soon.
         # This method will generate metadata from the sample data.
         # Additionally, the sample data will be employed to estimate the ndv (the number of distinct values) and cardinality.
-        #raise NotImplementedError
```

Comment:
I removed this comment because it seems like the sampling method was just recently implemented